ui/game/src/view/status.ts

```diff
@@ -73,7 +73,7 @@ export default function status(ctrl: Ctrl): string {
       if (insufficientMaterial(d.game.variant.key, d.game.fen))
         return `${i18n.site.insufficientMaterial} • ${i18n.site.draw}`;
```

Comment:
it seems like we already have a reusable translation there.

Reply:
@ornicar Updated to use it, but having 'Draw claimed' as well avoids confusion (since in the other cases the draw is automatic).